schemas/node.json

```diff
@@ -26,7 +26,12 @@
       },
       "minItems": 0,
       "uniqueItems": true
+    },
+    "status": {
+      "description": "Speaks about the stage of interaction with the reader.",
+      "type": "string",
```

Comment:
same, I would not add this yet (and definitely not to this model), let's refer this commit

Reply:
`+`